src/datahike/middleware/query.cljc

```diff
@@ -0,0 +1,10 @@
+(ns datahike.middleware.query
+  (:require [taoensso.timbre :as log]))
+
+(defn timed [query-handler]
```

Comment:
We might want to change this function name to `timed-query` or something similar, because `timed` is a macro in some of our benchmarking namespaces, so this might lead to confusions.